src/crawlers.ts

```diff
@@ -97,7 +88,12 @@ async function createAndStartSearchCrawler(
             let rank = 1;
             for (const result of results) {
                 result.rank = rank++;
-                const r = createRequest(result, responseId, request.userData.timeMeasures!);
+                const r = createRequest(
+                    result,
+                    responseId,
+                    request.userData.playwrightScraperSettings!,
+                    request.userData.timeMeasures!,
+                );
```

Comment:
I would refactor this more in future PR, at least it should have named parameters, this is already hard to order. But let's not do it in this PR